kube-runtime/src/reflector/object_ref.rs

```diff
@@ -2,14 +2,102 @@ use derivative::Derivative;
 use k8s_openapi::{api::core::v1::ObjectReference, apimachinery::pkg::apis::meta::v1::OwnerReference};
 use kube_client::{
     api::{DynamicObject, Resource},
-    core::ObjectMeta,
-    ResourceExt,
+    core::{api_version_from_group_version, ObjectMeta},
 };
 use std::{
+    borrow::Cow,
     fmt::{Debug, Display},
     hash::Hash,
 };
 
+/// An object that can be indexed by a [reflector store](super::Store).
+///
+/// This trait is blanket-implemented for all [`Resource`] objects.
+#[allow(clippy::module_name_repetitions)]
```

Comment:
This file should be renamed to `lookup` now, surely?

Reply:
This file contains both `ObjectRef` and `Lookup`, so whichever name to take is fine since it's not exported anyway. I assume you prefer to name the entire file lookup.rs instead of just deleting this `#[allow]`?

Reply:
Ah, you are right, sorry. No need to change it.